client/src/components/ProfileDropdown/ProfileDropdown.jsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 import { Dropdown } from 'semantic-ui-react';
 import { connect } from 'react-redux';
-import * as actions from '../store/actions/index';
+import * as actions from '../../store/actions/index';
+import styles from './ProfileDropdown.module.css';
 
 const ProfileDropdown = ({ onLogout }) => (
-  <Dropdown text="Profile" item floating style={{ color: 'white' }}>
-    <Dropdown.Menu>
+  <div className={styles.container}>
+  <Dropdown text="Profile" item floating>
+    <Dropdown.Menu direction={"left"}>
       <Dropdown.Item text="Profile" />
-      <Dropdown.Item text="My blogs" />
       <Dropdown.Item text="Sign out" onClick={onLogout} />
     </Dropdown.Menu>
   </Dropdown>
+  </div>
 );
 
```

Comment:
Why did you move this file?

Reply:
I like to have css and jsx in a same folder that's why.